cmd/formatter/logs.go

```diff
@@ -167,15 +170,15 @@ type logDecorator struct {
 	After     func()
 }
 
-func (l logDecorator) Log(containerName, message string) {
+func (l logDecorator) Err(containerName, message string) {
 	l.Before()
-	l.decorated.Log(containerName, message)
+	l.decorated.Err(containerName, message)
 	l.After()
 }
 
-func (l logDecorator) Err(containerName, message string) {
+func (l logDecorator) Log(containerName, message string) {
 	l.Before()
-	l.decorated.Err(containerName, message)
+	l.decorated.Log(containerName, message)
```

Comment:
🤔

Reply:
seems I used goland shortcut to move func up and down in file by mistake and didn't even noticed ;)